scripts/promise_batch_imports.py

```diff
@@ -15,23 +15,33 @@
 """
 
 from __future__ import annotations
+from typing import TYPE_CHECKING
 import requests
 import logging
+import re
 
 import _init_path  # Imported for its side effect of setting PYTHONPATH
 from infogami import config
 from openlibrary.config import load_config
 from openlibrary.core.imports import Batch
 from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
 
+if TYPE_CHECKING:
+    from re import Pattern
+
 logger = logging.getLogger("openlibrary.importer.promises")
 
 
-def format_date(date: str) -> str:
-    y = date[0:4]
-    m = date[4:6]
-    d = date[6:8]
-    return f"{y}-{m}-{d}"
+USE_ONLY_YEAR = re.compile(r"\d{4}(0101|0000)")
+
+
```

Comment:
```suggestion
```

Reply:
Oh whoops missed this can you remove and its imports?

Reply:
Whoops! Sorry. :(